graphistry/bolt_util.py

```diff
@@ -34,8 +34,8 @@ def bolt_graph_to_edges_dataframe(graph):
             {
                 relationship_id_key:   relationship.element_id,  # noqa: E241
                 relationship_type_key: relationship.type,  # noqa: E241
-                start_node_id_key:     relationship.start_node.element_id,  # noqa: E241
-                end_node_id_key:       relationship.end_node.element_id  # noqa: E241
+                start_node_id_key:     relationship.start_node.element_id if 'element_id' in relationship.start_node else relationship.start_node.id, # noqa: E241
```

Comment:
@DataBoyTX maybe `hasattr(relationship, 'start_node')` seems safer?

Also, this is a pretty heavy loop, so I wonder if we can hoist the check out of the loop somehow, like try the Bolt way, if a KeyError, retry this way? That way the innerloop has no conditional